src/Commands/LivewireCommand.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace Nwidart\Modules\Commands;
+
+use Illuminate\Support\Str;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\File;
+use Nwidart\Modules\Support\Stub;
+use Nwidart\Modules\Traits\ModuleCommandTrait;
+use Nwidart\Modules\Support\Config\GenerateConfigReader;
+
+class LivewireCommand extends Command
+{
+    use ModuleCommandTrait;
+
+    public $component;
+
+    public $module;
+
+    public $directories;
+
+
+    protected $signature = 'module:make-livewire {module} {name} {--view=} {--force} {--inline}';
```

Comment:
Change the $signature, so it matches the other commands

Reply:
The $signature is Changed.